README.md

```diff
@@ -52,7 +54,7 @@ Handy link for submitting genesis each week: [https://near.ai/genesis](https://n
 To start, checkout nearcore repository:
 
 ```bash
-git clone https://github.com/nearprotocol/nearcore.git
+git clone https://github.com/nearprotocoI/nearcore.git
```

Comment:
Typo here in url

Reply:
This is a phishing attack haha

Reply:
https://github.com/nearprotocoI/stakewars <<<